src/plugins/CoreLoadPlugin.ts

```diff
@@ -83,7 +83,17 @@ export default class DojoLoadPlugin {
 			compilation.moduleTemplate.plugin('module', (source: any, module: any) => {
 				if (module.meta && module.meta.isPotentialLoad) {
 					const path = stripPath(basePath, module.userRequest);
-					const require = `var require = function () { return '${path}'; };`;
+					const require = `var require = (function () {
+						var globalScope = typeof window === 'undefined' ? global : window;
```

Comment:
Normalizing the usage definitely seems like a good idea. I've updated the usage here to match the loader example, since that was more or less equivalent to what this was already doing but more thorough.